cps/static/js/table.js

```diff
@@ -93,6 +93,116 @@ $(function() {
         var domainId = $(e.relatedTarget).data("domain-id");
         $(e.currentTarget).find("#btndeletedomain").data("domainId", domainId);
     });
+
+    $('#restrictModal').on('hidden.bs.modal', function () {
+        // Destroy table and remove hooks for buttons
+        $("#restrict-elements-table").unbind();
+        $('#restrict-elements-table').bootstrapTable('destroy');
+        $("[id^=submit_]").unbind();
+        $('#h1').addClass('hidden');
+        $('#h2').addClass('hidden');
+        $('#h3').addClass('hidden');
+        $('#h4').addClass('hidden');
+    });
+    function startTable(type){
+        var pathname = document.getElementsByTagName("script"), src = pathname[pathname.length-1].src;
+        var path = src.substring(0,src.lastIndexOf("/"));
+        $("#restrict-elements-table").bootstrapTable({
+            formatNoMatches: function () {
+                return "";
+            },
+            url: path + "/../../ajax/listrestriction/" + type,
+            rowStyle: function(row, index) {
+                console.log('Reihe :' + row + ' Index :'+ index);
+                if (row.id.charAt(0) == 'a') {
+                    return {classes: 'bg-primary'}
+                }
+                else {
+                    return {classes: 'bg-dark-danger'}
+                }
+            },
+            onClickCell: function (field, value, row, $element) {
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: ['$element' is defined but never used.](https://app.codacy.com/manual/Ozzie-IsaacsOrganization/calibre-web/pullRequest?prid=4846308)